groups/alteryx-live/locals.tf

```diff
@@ -17,7 +17,12 @@ locals {
   secrets                     = data.vault_generic_secret.secrets.data
   alteryx_worker_ami_id       = var.alteryx_worker_ami_id == "" ? data.aws_ami.alteryx_worker_ami[0].id : var.alteryx_worker_ami_id
   alteryx_worker_ami_owner_id = local.account_ids_secrets["shared-services"]
-  alteryx_subnet_id           = [for o in data.aws_subnet.alteryx : o.id if o.tags["Name"] == "sub-alteryx-a" || o.tags["Name"] == "sub-alteryx-b"]
+  alteryx_subnet_id           = [for o in data.aws_subnet.alteryx : o.id if o.tags["Name"] == "sub-alteryx-b" || o.tags["Name"] == "sub-alteryx-c"]
```

Comment:
As you're now using specific subnets, this can be replaced with a single data lookup for the specific subnet in question. E.g. `data.aws_subnet.alteryx.id`